lib/EnsEMBL/REST/Model/Lookup.pm

```diff
@@ -148,9 +149,8 @@ sub find_compara_species_sets {
   my $mlsss = $compara_dba->get_MethodLinkSpeciesSetAdaptor->fetch_all_by_method_link_type($method);
   foreach my $mlss (@$mlsss) {
     my $species_set = {};
-    my $species_set_obj = $mlss->species_set_obj();
-    my @species_set_genomes = map { $_->name } @{$species_set_obj->genome_dbs};
-    $species_set->{species_set_group} = $species_set_obj->name;
+    my @species_set_genomes = map { $_->name } @{$mlss->species_set->genome_dbs};
```

Comment:
would it be worth keeping the same structure as before and create a species_set variable that is then used in both calls?